ext/web/lib.deno_web.d.ts

```diff
@@ -223,22 +231,38 @@ declare type EventListenerOrEventListenerObject =
   | EventListener
   | EventListenerObject;
 
-/** @category Events */
+/**  adds three more optional properties to those inherited from
+ * EventListenerOptions.
+ *
+ * @category Events */
 declare interface AddEventListenerOptions extends EventListenerOptions {
-  once?: boolean;
+  /** When set to true, indicates that the event listener should be automatically removed after its first invocation. */
   passive?: boolean;
+  /** When set to true, signals to the browser that the event listener will not call `preventDefault()`. */
```

Comment:
```suggestion
  /** When set to true, signals to the dispatcher that the event listener will not call `preventDefault()`. */
```

Also this should be on `passive`